elfpy/data/postgres.py

```diff
@@ -302,6 +339,46 @@ def get_pool_info(session: Session, start_block: int | None = None, end_block: i
     return pd.read_sql(query.statement, con=session.connection()).set_index("blockNumber")
 
 
+def get_checkpoint_info(session: Session, start_block: int | None = None, end_block: int | None = None) -> pd.DataFrame:
+    """Gets all info associated with a given checkpoint.
+
+    This includes
+    - `sharePrice` : The share price of the first transaction in the checkpoint.
+    - `longSharePrice` : The weighted average of the share prices that all longs in the checkpoint were opened at.
+    - `shortBaseVolume` : The aggregate amount of base committed by LPs to pay for bonds sold short in the checkpoint.
+
+    Arguments
+    ---------
+    session : Session
+        The initialized session object
```

Comment:
The docstring here doesn't seem to match the function signature

Reply:
ruff concurs: `elfpy/data/postgres.py:99:5: D417 Missing argument description in the docstring: `table_name``